orc/src/test/java/org/apache/iceberg/orc/TestTableProperties.java

```diff
@@ -53,12 +56,20 @@
 
   @Test
   public void testOrcTableProperties() throws Exception {
-    Long stripeSizeBytes = 32L * 1024 * 1024;
-    Long blockSizeBytes = 128L * 1024 * 1024;
```

Comment:
Changed `Long` to `long` because what `OrcConf.XXX#getLong` returns is a `long`, and I got 

> Ambiguous method call. Both assertEquals(Object, Object) in Assert and assertEquals(long, long) in Assert match

when using both `Long` and `long` as parameters for `Assert#assertEquals`.